### ISSUE TITLE
Doesn't work on Edge

### ISSUE THREAD

**Issue by doom777** [fireship-io/fireship.io#7]
Most of the website doesn't work on Edge (not everyone uses Chrome...)

**Comment by codediodeio** [comment]
You're right. The polyfill needs to be added for the web components to work. This will be addressed ASAP.

**Comment by codediodeio** [comment]
The polyfill for custom elements is fine, but I'm not sure how to support shadow dom. Many of the components use native slots `<slot>`, which is a problem on browsers without shadow dom support. https://github.com/angular/angular/issues/23636#issuecomment-401556727
